packages/web/config/chain-infos.ts

```diff
@@ -3598,6 +3598,34 @@ const mainnetChainInfos: SimplifiedChainInfo[] = [
     features: ["ibc-transfer", "ibc-go"],
     explorerUrlToTx: "https://ping.pub/gitopia/tx/{txHash}",
   },
+  {
+    rpc: "https://pirin-cl.nolus.network:26657",
+    rest: "https://pirin-cl.nolus.network:1317",
+    chainId: "pirin-1",
+    chainName: "Nolus",
+    bip44: {
+      coinType: 118,
+    },
+    bech32Config: Bech32Address.defaultBech32Config("nolus"),
+    currencies: [
+      {
+        coinDenom: "NLS",
+        coinMinimalDenom: "unls",
+        coinDecimals: 6,
+        coinGeckoId: "usd-coin",
+        coinImageUrl: "/tokens/nolus.svg",
```

Comment:
I don't see this image uploaded anywhere

Reply:
It is already there form an old PR
[packages/web/public/tokens/nolus.svg](https://github.com/osmosis-labs/osmosis-frontend/blob/master/packages/web/public/tokens/nolus.svg)